lib/Dialect/FIRRTL/Transforms/LowerBundleVectorTypes.cpp

```diff
@@ -896,7 +896,7 @@ void TypeLoweringVisitor::visitDecl(InstanceOp op) {
   // FIXME: annotation update
   auto newInstance = builder->create<InstanceOp>(
       resultTypes, op.moduleNameAttr(), op.nameAttr(), op.annotations(),
-      builder->getArrayAttr(newPortAnno));
+      builder->getArrayAttr(newPortAnno), builder->getBoolAttr(false));
```

Comment:
should this just pass the old instance's lowerToBind Attr?

Reply:
Yes. Whoops. Good catch.